### PR TITLE
Add Disabled to New-UDInputField

### PR DESCRIPTION
**Pull request by mattselle**
https://github.com/ironmansoftware/universal-dashboard/issues/1380

### CODE REVIEW

File: src/UniversalDashboard/Cmdlets/Inputs/NewInputFieldCommand.cs [hunk 1]
```diff
@@ -27,6 +27,9 @@ public class NewInputFieldCommand : PSCmdlet
         [Parameter()]
         public string[] Placeholder { get; set; }
 
+        [Parameter()]
+        public SwitchParameter Disabled { get; set; } = false;
```
Comment (adamdriscoll) [inline src/UniversalDashboard/Cmdlets/Inputs/NewInputFieldCommand.cs]: You don't need to set this to false. I also don't know what setting to false will do. If it's working, probably nothing.

**Comment by adamdriscoll** [comment]
One nit but overall this looks great!

File: src/UniversalDashboard/Cmdlets/Inputs/NewInputFieldCommand.cs (hunk 1 above)
Comment (mattselle) [reply src/UniversalDashboard/Cmdlets/Inputs/NewInputFieldCommand.cs]: That was a left over thing from me having it a bool and not a switch

**Comment by adamdriscoll** [comment]
This looks good. Thanks!

**Pull request merged**